test/e2e/common/networking.go

```diff
@@ -31,9 +31,10 @@ var _ = Describe("[sig-network] Networking", func() {
 		// expect exactly one unique hostname. Each of these endpoints reports
 		// its own hostname.
 		/*
-			    Testname: networking-intra-pod-http
-			    Description: Try to hit test endpoints from a test container and make
-				sure each of them can report a unique hostname.
+			Release : v1.9
+			Testname: Networking, intra pod http
+			Description: Create a hostexec pod that is capable of curl to netcat commands. Create a test Pod that will act as a webserver front end exposing ports 8080 for tcp and 8081 for udp. The netserver service proxies are created on specified number of nodes.
+			The kubectl exec on the webserver container MUST reach a http port on the each of service proxy endpoints in the cluster and the request must be successful. Container will execute curl command to reach the service port within specified max retry limit and should result in reporting unique hostnames.
```

Comment:
for all the test cases in here: there's an uncapitalized MUST, and there's a should not in all caps that I'm guessing needs to be a MUST as well

Reply:
Fixed.